app/models/video.py

```diff
@@ -0,0 +1,25 @@
+from flask import current_app
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.schema import ForeignKey
+from sqlalchemy import Table, Column, Integer, ForeignKey # OR line2?
+from app import db
+from app.models.customer import Customer
+
+
+class Video(db.Model):
+    __tablename__="videos"
+
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True, default=None) # default=None correct?
+    total_inventory = db.Column(db.Integer, default=0)  
+    
```

Comment:
Like with `Customer` you can compute `available_inventory` with a relationship:

```suggestion
    rentals = db.relationship('Rental', back_populates='video', lazy=True)
```